portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -26,25 +26,34 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/comment")
 public class CommentServlet extends HttpServlet {
-
   private final List<String> comments = new ArrayList<>();
 
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
     String json = new Gson().toJson(comments);
     response.getWriter().println(json);
   }
+
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String commenttext = getComment(request);
-    comments.add(commenttext);
+    String commentText = getComment(request);
+    comments.add(commentText);
     response.sendRedirect("/index.html");
   }
+
   private String getComment(HttpServletRequest request) {
-    String comment = request.getParameter("comment");
-    return comment;
+    try{
+      if(request.getParameter("comment") == null){
+        throw new Exception("Wrong Parameter path"); 
+      }
+      String comment = request.getParameter("comment");
+      return comment;
+    }catch(Exception path){
+      String comment = "Please check your code, because the path to get the comment is wrong";
+      return comment;
```

Comment:
You're mixing two techniques here: if, and try/catch.

Reply:
I fixed the code like we discussed